cpp/include/Ice/Communicator.h

```diff
@@ -13,11 +13,14 @@
 #include "Initialize.h"
 #include "PluginF.h"
 #include "Properties.h"
-#include "ProxyF.h"
+#include "Proxy.h"
```

Comment:
My cleanup of the headers included by Properties.h revealed a bug in this include list.